Source/Icebreaker/Helpers/IcebreakerBotDataProvider.cs

```diff
@@ -67,6 +69,57 @@ public async Task UpdateTeamInstallStatusAsync(TeamInstallInfo team, bool instal
             }
         }
 
+        /// <summary>
+        /// Get a list of past pairings
+        /// </summary>
+        /// <returns>List of past pairings.</returns>
+        public async Task<IList<PairInfo>> GetPairHistoryAsync()
+        {
+            await this.EnsureInitializedAsync();
+
+            var pairHistory = new List<PairInfo>();
+
+            try
+            {
+                using (var lookupQuery = this.documentClient
```

Comment:
This fetches all the pairings ever made. Wouldn't the data fetched here grow w/o bound?

Reply:
Yes. We have plans to expand this functionality to allow users to choose the time period (1,2,3,... pairing phases) that they want to avoid repeated pairings for. Most likely we will put a maximum on this, and at that point we can clean out some data/limit the data fetched.